test/e2e/v1/e2e_suite.go

```diff
@@ -120,7 +121,8 @@ var _ = Describe("kubebuilder", func() {
 				podOutput, err := kbc.Kubectl.Get(
 					true,
 					"pods", "-l", "control-plane=controller-manager",
-					"-o", "go-template={{ range .items }}{{ if not .metadata.deletionTimestamp }}{{ .metadata.name }}{{ \"\\n\" }}{{ end }}{{ end }}",
+					"-o", "go-template={{ range .items }}"+
+						"{{ if not .metadata.deletionTimestamp }}{{ .metadata.name }}{{ \"\\n\" }}{{ end }}{{ end }}",
```

Comment:
It would make more sense to put the last `{{ end }}` on a separate line since it delimits the `{{ range .items }}` pipeline.